src/User32.Desktop/User32+ScanCode.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the Keyboard scan code constants.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// General keyboard scan code constants on the same order that it can be found on <see cref="VirtualKey" /> constants
+        /// </summary>
+        /// <remarks>Scan codes are device-dependant values, thesse are general values used by most keyboards.</remarks>
```

Comment:
I cleaned the zeroes. About the values. They are device-dependant but there are [values that work for most of the default keyboards](http://terpconnect.umd.edu/~nsw/enpm808c/scancode.htm), these are the values that I entered on the enum (assuming a default international keyboard is used).

Scan codes can be AT or XT, depending on the keyboard hardware and layout. Here are some guidelines from Microsoft for [Scan Codes on WIndows CE 5.0](https://msdn.microsoft.com/en-us/library/ms892480.aspx), it applies to regular Windows versions as well.

There is a good article on that here: [Scan Codes Demistified](http://www.quadibloc.com/comp/scan.htm)

P.S.: [PInvoke.net](http://www.pinvoke.net/default.aspx/Structures/KEYBDINPUT.html) has the basis that I used to do that